### PR TITLE
Fixes #25465 - limit load macros in preview mode

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Review by lzap — APPROVED** [review]
Looks sane.

File: lib/foreman/renderer/scope/macros/base.rb [hunk 1]
```diff
@@ -109,11 +109,14 @@ def validate_subnet(subnet)
           #   .each { |batch| batch.each { |record| record.name }}
           # or
           #   .each_record { |record| record.name }
-          def load_resource(klass:, search:, permission:, batch: 1_000, includes: nil)
+          def load_resource(klass:, search:, permission:, batch: 1_000, includes: nil, limit: nil)
+            limit ||= 10 if preview?
```
Comment (lzap) [inline lib/foreman/renderer/scope/macros/base.rb:113]: However only ten seems to be bit of an overkill, how about 100?

**Comment by lzap** [comment]
Whatever, merging thanks.

**Comment by tbrisker** [comment]
1.20 - c1ab9673a

**Pull request merged**